plugins/jobtype/src/azkaban/jobtype/javautils/FileUtils.java

```diff
@@ -0,0 +1,47 @@
+package azkaban.jobtype.javautils;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.log4j.Logger;
+
+
+public class FileUtils {
+  private static Logger logger = Logger.getLogger(FileUtils.class);
+
+  /**
+   * Delete file or directory.
+   * (Apache FileUtils.deleteDirectory has a bug and is not working.)
```

Comment:
Just tried it and still not working although we are using the latest version.

Exception in thread "Thread-14" java.lang.IllegalArgumentException: /home/jnchang/workspace/azkaban-connector/deployed/azkaban-solo-server-2.7.0/unjar_tdch/com/teradata/connector/common/exception is not a directory